service/internal/featuretest/s3/presign_test.go

```diff
@@ -0,0 +1,180 @@
+package unit_test
```

Comment:
I think this would be better suited as a unit test inside of the relevant service, e.g. `service/s3/internal/customizaions/presign_test.go`

Each one of the presign unit tests that we add will need to be their own file anyways, so might as well keep it with the service instead of a separate module.